vivarium/core/engine.py

```diff
@@ -424,12 +424,11 @@ def __init__(
                 provide as the value for the key ``experiment_id``.
             display_info: prints experiment info
             progress_bar: shows a progress bar
-            store_emit: An optional dictionary to turn emits on or off. This
-                dictionary may contain the keys (`on`,`off`), mapping to a list
-                of paths in the Store hierarchy to be turned on or off. The
-                on configs take precedence over the off configs in that all
-                paths in `off` are turn off first, and can be turned on again
-                by the paths in `on`.
+            store_schema: An optional dictionary to expand the store hierarchy
+                configuration, and also to turn emits on or off. The dictionary
+                needs to be structured as a hierarchy, which will expand the
+                existing store hierarchy. Setting an emit value for a branch
+                node will set the emits of all the leaves to that value.
```

Comment:
So technically I think this is a breaking change since we're removing the `store_emit` flag which has already been released. It's coming so close that I doubt anyone has started using `store_emit` yet, but I think we should try to avoid these kinds of flip-flops in the future so that we don't break our API too often